dispatcher.c

```diff
@@ -41,6 +41,7 @@
 #endif
 #ifdef HAVE_LZ4
 #include <lz4.h>
+#include <lz4frame.h>
```

Comment:
This needs an associated change to configure.ac to check ls4frame.h is available.  Check the with_ssl check, it also checks two headers (err.h and ssl.h), I think it should be possible to do the same for with_lz4 here.

Reply:
done